core/conversion/converters/impl/element_wise.cpp

```diff
@@ -484,8 +484,12 @@ auto element_wise_registrations TORCHTRT_UNUSED =
         .pattern({"aten::ne.Scalar(Tensor self, Scalar other) -> (Tensor)",
                   [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
                     auto self = args[0].ITensorOrFreeze(ctx);
-                    auto scalar = args[1].unwrapToScalar().to<float>();
-                    auto scalar_tensor = tensor_to_const(ctx, torch::tensor({scalar}));
+                    auto scalar = args[1].unwrapToScalar();
+                    nvinfer1::ITensor* scalar_tensor;
+                    if (scalar.isFloatingPoint())
```

Comment:
Just a code style thing, would prefer if you have the brackets here

Reply:
Ok Naren, I will add brackets here.